README.md

```diff
@@ -290,6 +290,7 @@ Double-chord shortcuts:
 | `P`          | push     | **Push**             |
 | `R`          | requests | **Pull Requests**    |
 | `U`          | update   | **Update**           |
+| `Y`          | why      | **Annotate**         |
```

Comment:
Another option is to use <kbd>W</kbd> for "who/why"

Reply:
Thank you!